src/SelectableBox/tests/SelectableBoxSet.test.jsx

```diff
@@ -68,10 +68,13 @@ describe('<SelectableBox.Set />', () => {
       expect(screen.getByTestId('checkbox-set')).toBeInTheDocument();
     });
     it('renders with radio type if neither checkbox nor radio is passed', () => {
+      // eslint-disable-next-line no-console
```

Comment:
Maybe just disable this rule for the whole file by adding `// eslint-disable no-console` at the top of the file instead of repeating this line multiple times?

Reply:
Done